### PR TITLE
clearInterval timer on dispose()

### PR DESCRIPTION
**Pull request by mbieser**
https://github.com/waldophotos/kafka-avro/issues/60

### CODE REVIEW

**Comment by ricardohbin** [comment]
Thanks. Can you put yourself as contributor in changelog?

**Comment by mbieser** [comment]
@ricardohbin  Done.

File: package.json [hunk 1]
```diff
@@ -13,6 +13,9 @@
     {
       "name": "Ricardo Bin",
       "email": "ricardohbin@gmail.com"
+    },
+    {
+      "name": "mbieser"
```
Comment (ricardohbin) [inline package.json]: this is a core commiter list, if everyone who contributed once the list would be too big :/

**Comment by ricardohbin** [comment]
@mbieser just one point in package.json and I bump it

File: package.json (hunk 1 above)
Comment (mbieser) [reply package.json]: Removed.

**Comment by ricardohbin** [comment]
Thank you! `1.1.4` is ready

**Pull request merged**